i18n/fi_FI.csv

```diff
@@ -68,21 +68,21 @@ Shipping,Toimitus
 "Please navigate to your Magento orders, select the order in question, and click restore order. Note the stock.", "Ole hyvä, ja valitse kyseinen tilaus Magenton hallinnassa ja klikkaa Palauta tilaus. Huomioi varastosaldo."
 "Restore Order", "Palauta tilaus"
 "Are you sure you want to make changes to this order?", "Haluatko varmasti tehdä muutoksia tähän tilaukseen?"
-"Pending payment from OP Payment Service","Odottaa maksua OP Payment Servicestä"
-"Order canceled. Failed to redirect to OP Payment Service.","Tilaus peruutettu. Ohjausta OP Payment Serviceen ei voitu tehdä"
-"Op Payment Service API credentials are incorrect. Please contact support.","Op Payment Service tunnuksia ei löytynyt. Ole hyvä, ja ota yhteyttä asiakaspalveluun."
+"Pending payment from Checkout Finland","Odottaa maksua Checkout Finlandistä"
```

Comment:
Odottaa maksua Checkout Finlandista

Is this correct @loueranta

Reply:
@richard-madarasz could you still change:

"Odottaa maksua Checkout Finlandistä" to "Odottaa maksua Checkout Finlandista"